### PR TITLE
Add image for slurm-docker-cluster

### PR DESCRIPTION
**Pull request by philippotto**
This image is used by the clustertools CI which runs tests within a slurm context.

### CODE REVIEW

**Comment by philippotto** [comment]
The files were copied from https://github.com/scalableminds/cluster_tools.

**Review by normanrz — APPROVED** [review]

**Comment by normanrz** [comment]
I also created the dockerhub repo `scalableminds/slurm-docker-cluster`

File: slurm-docker-cluster/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Slurm Docker Cluster
+
+This is a multi-container Slurm cluster using docker-compose.  The compose file
+creates named volumes for persistent storage of MySQL data files as well as
+Slurm state and log directories.
```
Comment (jstriebel) [inline slurm-docker-cluster/README.md:5]: There's no more docker-compose file in this folder anymore, should we link to the webknossos-libs repo?

Comment (jstriebel) [reply slurm-docker-cluster/README.md:5]: cc @philippotto

Comment (philippotto) [reply slurm-docker-cluster/README.md:5]: Good point. I just pushed https://github.com/scalableminds/dockerfiles/commit/01d1f9d1f1c34f20df167f3abad058efd9eff4f0.

**Pull request merged**